doc/source/output/json.rst

```diff
@@ -1,12 +1,11 @@
-
-.. program::  gcovr
-
 .. _json_output:
 
 JSON Output
 ===========
 
-The ``gcovr`` command can also generate a JSON output using
+.. program:: gcovr
```

Comment:
Why moving this in between the text?

Reply:
Just for consistency with other files, e.g., `clover.rst`

Reply:
Tier it is between heading and text which looks better in your opinion.